sopel/tools/target.py

```diff
@@ -1,16 +1,15 @@
+"""Trigger's targets: users and channels."""
```

Comment:
Not much to do with triggers. These objects are mostly used for keeping track of which users and channels (plus privileges) Sopel knows about (in `bot.channels` and `bot.users`).

```suggestion
"""User and channel objects used in state tracking."""
```